web/src/components/tutorials.astro

```diff
@@ -182,8 +187,8 @@ const tutorials = [
     link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/ml-to-fl"
   },
   {
-    title: "Hello FedAvg Numpy",
-    tags: ["beg.", "algorithm", "numpy", "model-controller"],
+    title: "Hello FedAvg",
+    tags: ["beg.", "algorithm", "pytorch", "model-controller"],
     description: "Example using the FedAvg workflow to implement Federated Averaging.",
     link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/hello-fedavg/README.md"
```

Comment:
no related to this PR, maybe there is somewhere we could use variable substitution or something for these stuff?
<github link [2.5]> or something.

Reply:
yes good idea, I can later look to add substitution for the links and automatic versioning for the site content